src/topology/algebra/TopCommRing/basic.lean

```diff
@@ -3,8 +3,8 @@ Copyright (c) 2019 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
-import algebra.CommRing.basic
-import topology.Top.basic
+import algebra.comm_ring.category
```

Comment:
Logically shouldn't this file also be renamed to `topology/algebra/topological_commutative_ring/category/basic.lean`?

Reply:
Yes, I wasn't sure what to do here, and forgot to mention it as an open problem. :-)